README.rst

```diff
@@ -138,28 +132,45 @@ The plugin offers some integrations listed below:
                profiles_sample_rate: 0.5
             another_client_parameter: 'value'
 
-
 Auditing Django views
 =====================
 
-The majority of views in eox-core use an auditing decorator, defined in our custom library called `eox-audit-model`_,
-that helps saving relevant information about non-idempotent operations. By default this functionality is turned on. To
+Most views in eox-core use an auditing decorator, defined in our custom library, *eox-audit-model*,
+that helps save relevant information about non-idempotent operations. By default, this functionality is turned on. To
 check your auditing records go to Django sysadmin and find DJANGO EDUNEXT AUDIT MODEL.
 
-For more information, check the eox-audit-model documentation.
+You can check the `eox-audit-model`_ documentation for more information.
 
+Development
+===========
 
-.. _Open edX Devstack: https://github.com/edx/devstack/
-.. _openedx plugin: https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins
-.. _edx-platform: https://github.com/edx/edx-platform/
+Dependency Management
+---------------------
+
+EOX core now follows `OEP-18`_ so the correct way to update dependencies is to run ``make upgrade`` inside your virtual environment.
```

Comment:
Is this section needed? I haven't seen it in other repos.

Reply:
I didn't want to remove it without being sure if should keep it or not.

Reply:
I'd say we don't need for a few reasons:
1. It was added back in 2020 while updating eox-core: https://github.com/eduNEXT/eox-core/blame/1176b68864847c2d3eec412e69e4c718106c161f/README.rst#L101 
2. That's the current standard for upgrading packages in a plugin, I don't think we need to specify it here

Reply:
^ Non-blocking comment.